common/openapi.yml

```diff
@@ -223,6 +223,25 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/fenceAccountKey/v1/{provider}/key:
```

Comment:
if the api group is `fenceAccountKey`, do we really need the `/key` on the end?

Reply:
That's a good point! I agree it's redundant